starknet_devnet/blueprints/rpc.py

```diff
@@ -2,19 +2,27 @@
 RPC routes
 rpc version: 0.15.0
 """
-from __future__ import annotations
+# pylint: disable=too-many-lines
 
+from __future__ import annotations
+import dataclasses
 import json
-from typing import Callable, Union, List, Tuple, Optional, Any
 
+from typing import Callable, Union, List, Tuple, Optional, Any
 from typing_extensions import TypedDict
-
 from flask import Blueprint, request
+import marshmallow
```

Comment:
This should be made a dependency, we shouldn't be relying on a third party to install it.